cpu/stm32f0/periph/gpio.c

```diff
@@ -195,7 +202,7 @@ void isr_exti(void)
     for (size_t i = 0; i < EXTI_NUMOF; i++) {
```

Comment:
AFAIK this is not needed, because if you disable a line, the corresponding bit in the `EXTI->PR` is not set on any event...